system-variables.md

```diff
@@ -309,7 +322,7 @@ This variable is an alias for `last_insert_id`.
 - Scope: NONE
 - Type: Boolean
 - Default value: `OFF`
-- This variable indicates whether [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) is used.
+- This variable indicates whether [TiDB Binlog](https://docs.pingcap.com/tidb/stable/tidb-binlog-overview) is used.
```

Comment:
Since this is not wrapped by `<CustomContent platform="tidb-cloud">`, I don't think we need to change the link.

Reply:
If we do not update it, "/tidb-binlog/tidb-binlog-overview.md" will report a 404 error in TiDB Cloud docs because TiDB Cloud does not have this doc. Because TiDB Binlog is an independent tool, think it is okay to link to its latest doc?  Otherwise, we have to use CustomConent to provide two links, one external link for TiDB Cloud, and one internal link for TiDB.  Which one do you prefer?